google/cloud/storage/constants.py

```diff
@@ -106,6 +106,14 @@
 
 PUBLIC_ACCESS_PREVENTION_UNSPECIFIED = "unspecified"
 """Unspecified public access prevention value.
+DEPRECATED: Remove once change is fully complete in backend
+
+See: https://cloud.google.com/storage/docs/public-access-prevention
+"""
+
+PUBLIC_ACCESS_PREVENTION_INHERITED = "inherited"
+"""Inherited public access prevention value.
+Remove once change is fully complete in backend
 
 See: https://cloud.google.com/storage/docs/public-access-prevention
 """
```

Comment:
I don't understand the "Remove" bit -- this new constant should remain:
```suggestion
PUBLIC_ACCESS_PREVENTION_INHERITED = "inherited"
"""Inherited public access prevention value.

See: https://cloud.google.com/storage/docs/public-access-prevention
"""
```

Reply:
Oh nice catch, thanks